src/absil/il.fs

```diff
@@ -498,6 +501,7 @@ type ILScopeRef =
         | ILScopeRef.Local -> ""
         | ILScopeRef.Module mref -> "module "+mref.Name
         | ILScopeRef.Assembly aref -> aref.QualifiedName
+        | ILScopeRef.PrimaryAssembly -> ""
```

Comment:
Should a PrimaryAssembly not have a Qualified name?

Reply:
You can't determine the qualified name because you do not know what the primary assembly is at this point. But, that got me thinking - I should check to see how the qualified name is used.

Reply:
Ok, looks like I already checked before as I handled a case where qualified names were used in the code quotation translator for older F# 2.0 dlls.